### PR TITLE
Test to automate create-list-delete 1000 buckets and also list bucket operation for bucket names within range 3-63 chars long

### PR DESCRIPTION
**Pull request by kapiljinna**
Test to automate create-list-delete 1000 buckets and also list bucket operation for bucket names within range 3-63 chars long using the default user credentials .

Signed-off-by: Kapil Jinna <kapil.jinna@seagate.com>

# Problem Statement
Automate below tests using default user credentials:
- For creating-listing-deleting 1000 buckets .
- Verify list bucket operation for bucket names within range 3-63 chars long .

# Design
TEST JIRA ids to be automated :
-https://jts.seagate.com/browse/TEST-36100
-https://jts.seagate.com/browse/TEST-36157

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Review by seagate-sarang-sawant — APPROVED** [review]

File: comptests/s3/test_bkt_crud_operations_rgw.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        self.log.info(
+            "STARTED: Verification of max. no. of buckets(1000) user can create and then list and delete them")
+        self.log.info("Creating 1000 max buckets")
+        resp1 = self.s3_test_obj.bucket_count()
+        self.log.info(resp1)
+        bkt_list = []
+        for each in range(1000):
+            bucket_name = "{0}{1}".format("bkttest-36100-", each)
+            resp = self.s3_test_obj.create_bucket(bucket_name)
+            assert resp[0], resp[1]
```
Comment (DhobaleVishal) [inline comptests/s3/test_bkt_crud_operations_rgw.py:91]: Use assert_utils for any type of assert.

File: comptests/s3/test_bkt_crud_operations_rgw.py [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+            bkt_list.append(bucket_name)
+        self.log.info("Created 1000 buckets")
+        self.log.info("Verifying that bucket is created")
+        resp = self.s3_test_obj.bucket_count()
+        self.log.info(resp)
```
Comment (DhobaleVishal) [inline comptests/s3/test_bkt_crud_operations_rgw.py:96]: Avoid repeated logging as same was done in s3 libs.

File: comptests/s3/test_bkt_crud_operations_rgw.py [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+        assert resp[0], resp[1]
+        resp = self.s3_test_obj.delete_multiple_buckets(bkt_list)
+        self.log.info(resp)
+        assert_utils.assert_true(resp[0], resp1[1])
+        self.log.info("Verified that 1000 buckets are created")
+        self.log.info("ENDED: Verification of max. no. of buckets(1000) user can create and then list and delete them")
+
+    @pytest.mark.comp_s3
+    @pytest.mark.lc
```
Comment (DhobaleVishal) [inline comptests/s3/test_bkt_crud_operations_rgw.py:111]: I think this label no more required. Please remove it.

**Review by DhobaleVishal — CHANGES_REQUESTED** [review]
@kapiljinna Added inline comments. Please fix codacy issues.

**Comment by shailesh-vaidya** [comment]
Closing as an obsolete

**Pull request closed without merging**